cmd/template/cluster/flag.go

```diff
@@ -129,8 +133,6 @@ func (f *flag) Validate() error {
 				return microerror.Maskf(invalidFlagError, "--%s must be a valid subnet size (20, 21, 22, 23, 24 or 25)", flagControlPlaneSubnet)
 			}
 		}
-
-		return nil
```

Comment:
IMO this was a bug preventing rest of code to be executed (without reason)

Reply:
🤷